src/interfaces/index.tsx

```diff
@@ -77,3 +76,9 @@ export interface EditFormProps {
   course: Course;
   closeFormHandler: () => void;
 }
+
+export interface AddEditFormProps {
+  course: Course;
+  closeFormHandler: () => void;
+  type: string;
```

Comment:
Make a habit of arranging props/types/interfaces as 
1. state variables
2. other variables
3. functions

If possible (not necessary) in alphabetical order